src/AlphaVantage/StockDataHandlers.js

```diff
@@ -18,11 +18,11 @@ const fetchStockSymbols = async (searchQuery) => {
 
     return loadedStocks;
   } catch (error) {
-    console.log(error.message);
+    console.error(error);
```

Comment:
When propagating errors out to the console, it's better to log all the errors since it has some valuable information like the line number etc..